src/components/POLine/POLineDetails/FieldPaymentStatus.js

```diff
@@ -13,28 +14,53 @@ const PAYMENT_STATUS = {
   pending: 'Pending',
 };
 
+const PAYMENT_STATUSES_BY_ORDER_STATUS = {
+  Pending: [
```

Comment:
this looks like values from back-end, maybe it's better to have them as constants

Reply:
Thanks, I'll do it.